test/test_wrappers.jl

```diff
@@ -80,7 +80,7 @@ test_display(pointm, "Point{false, true}((1, 2, 3))", "Point((1,2,3))")
 pointm_crs = GI.Point((X=1, Y=2, M=3); crs=EPSG(4326))
 @test parent(pointm_crs) === parent(pointm)
 @test GI.crs(pointm_crs) === EPSG(4326)
-test_display(pointm_crs, "Point{false, true}((1, 2, 3), crs = EPSG{1}((4326,)))", "Point((1,2,3))")
+test_display(pointm_crs, "Point{false, true}((1, 2, 3), crs = \"EPSG:4326\")", "Point((1,2,3))")
```

Comment:
I wonder if showing EPSG as a string is confusing to users who still need to use the constructor

Reply:
Agreed, but it's equivalent in most cases that one might care about (Proj and GDAL)

Reply:
Maybe we should change show on GFT types to show the constructor too?